node_modules/oae-tenants/lib/rest.js

```diff
@@ -206,7 +207,8 @@ OAE.globalAdminRouter.on('post', '/api/tenantNetwork/:id/removeTenants', functio
  * @HttpResponse                    400             The tenant alias should not contain a space
  */
 OAE.globalAdminRouter.on('post', '/api/tenant/create', function(req, res) {
-    var opts = _.oaeExtendDefined({}, _.pick(req.body, 'emailDomain', 'countryCode'));
+    var opts = _.oaeExtendDefined({}, _.pick(req.body, 'countryCode'));
+    opts.emailDomains = OaeUtil.toArray(req.body.emailDomain);
```

Comment:
If our tenant UI parameter is `emailDomains`, then I think the query string parameter to set its value with should also be `emailDomains`, not `emailDomain`